wire-runtime/src/test/proto-java/com/google/protobuf/FileDescriptorProto.java

```diff
@@ -92,7 +92,7 @@ public FileDescriptorProto redact(FileDescriptorProto value) {
 
   public static final String DEFAULT_NAME = "";
 
-  public static final String DEFAULT_PACKAGE = "";
+  public static final String DEFAULT_PACKAGE_ = "";
```

Comment:
`package` is a keyword so we name the field `package_`. I've changed the thing that names the defaults to use the same policy, so it gets `DEFAULT_PACKAGE_`. It's a simpler rule, though I admit this particular case is worse.